tests/outputs/a_presentation/presentations.yaml

```diff
@@ -0,0 +1,39 @@
+2006as_mars:
+  abstract: the earth is round as seen from mars
+  authors: []
+  begin_date: 2020-06-26
+  end_date: 2020-06-26
+  location: Mars
+  meeting_name: flat earth
+  project:
+    - all
+  status: accepted
+  title: tbd
+  type: award
```

Comment:
please check allowed types from the schema and pick one for the test, declared at the CL, but in the code default to "invited" if it is not specified.   for the test maybe pick "contributed_oral" rather than award (award talks are rare!)

Reply:
done